extras/data/SMCSX/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+nnpdf_id: null
+fktable_id: null
+arxiv: https://inspirehep.net/literature/499139
+
+SMCSX_NC_17GEV_MUP_G1:
+  description: polarized structure function $g_1$ in SMCSX with proton target
```

Comment:
this is not a new experiment, right? but is SMC at small x - please reflect here

Reply:
done in df12f1fe72ec2eb91c0b265d7f35bff594ddaec1